hydrosdk/servable.py

```diff
@@ -140,3 +138,10 @@ def logs(self, follow=False):
             return sseclient.SSEClient(res).events()
         else:
             raise ServableException(f"{res.status_code} : {res.text}")
+
+    def predictor(self, return_type=PredictorDT.DICT_NP_ARRAY) -> PredictServiceClient:
```

Comment:
Internal services have cases when Servable needs to be used without monitoring.

Reply:
Fixed